npm/cypress-schematic/src/builders/cypress/index.ts

```diff
@@ -99,6 +99,14 @@ export function startDevServer ({
   devServerTarget,
   watch,
   context }: StartDevServerProps): Observable<string> {
+  // @NOTE: Do not forward watch option if not supported by the target dev server,
+  // this is relevant for running Cypress against dev server target that does not support this option,
+  // for instance @nguniversal/builders:ssr-dev-server.
+  // see https://github.com/nrwl/nx/blob/f930117ed6ab13dccc40725c7e9551be081cc83d/packages/cypress/src/executors/cypress/cypress.impl.ts
+  if (!devServerTarget.includes('watch')) {
```

Comment:
The [nrwl executor linked](https://github.com/nrwl/nx/blob/f930117ed6ab13dccc40725c7e9551be081cc83d/packages/cypress/src/executors/cypress/cypress.impl.ts#L163) checks the devServerTarget options, not the string itself. Most `devServerTarget` strings don't contain `watch` in the name, so this would disable watch for projects that do support it. We should use a similar API that Nx uses (`readTargetOptions`) to get the keys if Angular offers one.

Otherwise, an explicit hardcoding might be a quick fix for nguniversal if we can detect the executor/builder from the devServerTarget

Reply:
Hey  @ZachJW34 , 
first of all thanks for fast response. I tried to fix it, but I am super unsure about the changes. 
I tried to readout the options like nrwl, but this will bloat the builder a lot - therefore I went for option 2. 
Only add watch property if not universal ssr dev server as selected target. 

I also added a notification for the dev as hint that watch property will be added.